esm_runscripts/slurm.py

```diff
@@ -130,8 +129,9 @@ def calc_requirements(self, config):
         start_core = 0
         end_proc = 0 
         end_core = 0
+        path=self.path
```

Comment:
you can combine this into the line below.

```
return_things = self.mini_calc_reqs(self.path, config, ....)
```

Reply:
Probably a good sign to call it quits for the day xD